packages/components/src/Slider/Slider.component.js

```diff
@@ -0,0 +1,116 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { translate } from 'react-i18next';
+import RcSlider from 'rc-slider';
+import rcSliderTheme from 'rc-slider/assets/index.css'; // eslint-disable-line no-unused-vars
+
+import { Icon } from '../';
+import I18N_DOMAIN_COMPONENTS from '../constants';
+import { DEFAULT_I18N } from '../translate';
+import theme from './Slider.scss';
+
+function noOp() {}
+
+/**
+ * This function render an icon component
+ * @param {object} icon - Icons props ( selected & name )
+ * @param {number} key - index of the icon for the key
+ */
+function renderIcons(icon, key) {
+	return <Icon name={icon.name} className={icon.selected ? theme.selected : null} key={key} />;
+}
+
+/**
+ * This function give the selected icon position if there is more than 1 icon
+ * @param {array} icons - array of icons
+ * @param {number} value - current value of the slider
+ * @param {number} min - minimum value of the slider
+ * @param {number} max - maximum value of the slider
+ */
+export function getSelectedIconPosition(icons, value, min, max) {
+	if (icons && Array.isArray(icons) && icons.length > 1) {
+		const interval = (max - min) / (icons.length - 1);
+		const div = Math.floor(value / interval);
+		const rest = value % interval;
+		return rest < interval / 2 ? div : div + 1;
```

Comment:
The 3 lines above seems to be a Math.round()
```
const interval = (max - min) / (icons.length - 1);
return Math.round(value / interval);
```

Reply:
Oh shit, it's 100% a Math.round() 👍